siriuspy/siriuspy/devices/beamline.py

```diff
@@ -0,0 +1,70 @@
+"""Beamline devices."""
+
+from .device import Device as _Device
+
+
+class CAXCtrl(_Device):
+    """Carcara beamline control."""
+
+    PROPERTIES_DEFAULT = (
+        ':A:PP01:A', ':A:PP01:A.RBV',
+        ':A:PP01:B', ':A:PP01:B.RBV',
+        ':A:PP01:C', ':A:PP01:C.RBV',
+        ':A:PP01:D', ':A:PP01:D.RBV',
+        ':A:PP01:E', ':A:PP01:E.RBV',
+        )
+
+    def __init__(self, devname=None, props2init='all', **kwargs):
+        """Init."""
+        devname = 'CAX' if devname is None else devname
+        super().__init__(devname, props2init=props2init, **kwargs)
+
+    @property
+    def mirror_m1_pos_ry(self):
+        """Return mirror M1 linear position axis for Ry [mm]."""
+        return [':A:PP01::A.RBV']
+
+    @mirror_m1_pos_ry.setter
+    def mirror_m1_pos_ry(self, value):
+        """Set mirror M1 linear position axis for Ry [mm]."""
```

Comment:
I don't understand the docstrings. What do they mean?

Reply:
I updated docstrings. take a look